docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Extend_CMS_Interface.md

```diff
@@ -642,6 +642,146 @@ class MyGallery extends React.Component {
 export default withInjector(MyGallery);
 ```
 
+### Using injector to customise redux state data
+
+It is recommended to be familiar with the concepts of Immutability and Redux before starting this.
```

Comment:
good point, I've linked to https://facebook.github.io/immutable-js/ and http://redux.js.org/